src/Extensions/AzureBlobPayloads/PayloadStore/IPayloadStore.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.DurableTask;
+
+/// <summary>
+/// Abstraction for storing and retrieving large payloads out-of-band.
+/// </summary>
+public abstract class PayloadStore
```

Comment:
Can you make sure the filename and classname match? The filename implies it's an interface. Unless that's just github being weird, if so, disregard.